crates/uv-scripts/src/lib.rs

```diff
@@ -94,34 +106,11 @@ pub enum Pep723Error {
     Toml(#[from] toml::de::Error),
 }
 
-/// Read the PEP 723 `script` metadata from a Python file, if it exists.
-///
-/// See: <https://peps.python.org/pep-0723/>
-pub async fn read_pep723_metadata(
-    file: impl AsRef<Path>,
-) -> Result<Option<Pep723Metadata>, Pep723Error> {
-    let contents = match fs_err::tokio::read(file).await {
-        Ok(contents) => contents,
-        Err(err) if err.kind() == io::ErrorKind::NotFound => return Ok(None),
-        Err(err) => return Err(err.into()),
-    };
-
-    // Extract the `script` tag.
-    let Some(contents) = extract_script_tag(&contents)? else {
-        return Ok(None);
-    };
-
-    // Parse the metadata.
-    let metadata = toml::from_str(&contents)?;
-
-    Ok(Some(metadata))
-}
-
 /// Given the contents of a Python file, extract the `script` metadata block, with leading comment
-/// hashes removed.
+/// hashes removed and the python script.
```

Comment:
What is the "python script"? Can you show an example in the docstring here?

Reply:
By this I meant the python code in the rest of the script, I've added an example and some further explanation for this.